httpie/sessions.py

```diff
@@ -79,6 +81,12 @@ def update_headers(self, request_headers: RequestHeadersDict):
             if name == 'User-Agent' and value.startswith('HTTPie/'):
```

Comment:
This is pre=existing code, but please change it to use the same header name normalization like you use for `Cookie` bellow for consistency.